src/Firebase.js

```diff
@@ -119,20 +127,24 @@ export async function editSymptom(id, fogginess, anxiety, headache, fatigue, gut
         fatigue,
         gut,
         date,
+        userId: auth.currentUser.uid
```

Comment:
cara, to chocado que o set da um update em tudo...

quem sabe nao tem um "updateDoc"?